CHANGELOG.md

```diff
@@ -3,6 +3,208 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.0
+
+_Nov 10, 2023_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
```

Comment:
I'd also add a link to our v7 migration guide index page, do we have one?

```suggestion
We're thrilled to announce the first alpha release of our next major version, v7. 
This release introduces a few breaking changes, paving the way for the upcoming features like Pivoting and DateTimeRange.

A special shoutout to thank the 12 contributors who made this release possible. Here are some highlights ✨:
```

Reply:
We do not have a migration guide overview page, just single ones for each package. Should we introduce one?

Reply:
Nice idea worth discussing.
What would you think of putting there? 🤔